scripts/lighthouse/login.js

```diff
@@ -0,0 +1,15 @@
+async function main(browser, { url }) {
+  const page = await browser.newPage();
+
+  await page.goto(`${new URL(url).origin}/login`, { waitUntil: 'domcontentloaded' });
+  await page.type('[aria-label="Username input field"]', 'admin');
+  await page.type('[aria-label="Password input field"]', 'admin');
+  await page.click('[aria-label="Login button"]');
+  await Promise.all([
+    page.waitForNavigation(),
+    page.waitForSelector('[aria-label="Skip change password button"]').then(a => a.click()),
+  ]);
+  await page.close();
+}
+
+module.exports = main;
```

Comment:
Do we need to use puppeteer?
We moved away from pupeteer in favor of Cypress so I'm wondering if we can't execute a cypress test instead and collect the lighthouse data anyway?

Reply:
@hugohaggmark The simpler the better :+1: